webpack.config.js

```diff
@@ -31,6 +31,9 @@ module.exports = {
               minimize: true,
             },
           },
+          {
+            loader: 'sass-loader', // compiles Sass to CSS
```

Comment:
please re-write this comment to follow Airbnb comment style https://github.com/airbnb/javascript#comments